projects/addon-doc/src/components/code/code-block.ts

```diff
@@ -0,0 +1,4 @@
+export interface TuiCodeBlock {
```

Comment:
I thought we decided to drop this interface and rely on highlight.js figuring out type?

Reply:
fixed